doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+:banner: banners/extract_api.png
+:types: api
+
+:code-column:
+
+===============
+Extract API
+===============
+
+Odoo provides a service allowing you to automate the processing of your invoices. The service scans your document using an Optical Character Recognition engine (OCR) and then uses AI based algorithms to extract the fields of interest as the total, the due date or even the invoice lines. More information can be found on the `demo page <https://www.odoo.com/page/invoice-automation>`_.
```

Comment:
```suggestion
Odoo provides a service allowing you to automate the processing of your invoices. The service scans your document using an Optical Character Recognition (OCR) engine and then AI-based algorithms to extract the fields of interest such as the total, the due date, or the invoice lines. More information can be found on the `demo page <https://www.odoo.com/page/invoice-automation>`_.
```
This should be the other way around: the website page should link to the documentation for more information.

Reply:
Yes but I think it is good to have a link in this direction too.

Reply:
Maybe as advertising but the doc should always be the main source of information. The app pages are always prone to removal and are more difficult to maintain.